File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/elixir/valid.ex
# Fixture — café 日本語 before declarations
defmodule Fixture.Greeter do
  @moduledoc "A greeter."
  import String, only: [upcase: 1]

  @doc "Say hello."
  def hello(name), do: "hi " <> upcase(name)

  defp secret(x), do: x + 1

  defmacro shout(text) do
    quote do: unquote(text)
  end
end

File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/ocaml/valid.ml
(* Fixture — café 日本語 before declarations *)
open Printf

(** A point. *)
type point = { x : int; y : int }

(** Add two numbers. *)
let add a b = a + b

let limit = 10

module Inner = struct
  let nested x = x + 1
end

class greeter = object
  method hello name = "hi " ^ name
end

exception Failure_here of string
